### DISCUSSION TITLE
Screen size converter tool

### DISCUSSION THREAD

**Discussion by rlust** [HASwitchPlate/openHASP#203] [Q&A] [answered]
I am looking for ideas on how to convert the demo screen size (screen size 240x320) to (320x480) given in the examples.

Are there any tools for for calculating object placement by screen size.

Great project and thanks!

**Comment by nagyrobi** [comment, marked as answer]
You'll have to adjust them by hand. Multiply X values by 1.33, and Y values by 1.5 (or vice-versa if rotated). Round to integers.

Later a percentage value, might be adopted.

**Reply by rlust** [reply]
Thanks!
